contracts/RootChain.sol

```diff
@@ -109,6 +113,9 @@ contract RootChain {
       root: root,
       createdAt: block.timestamp
     });
+    ChildBlockCreated(currentChildBlock, root);
+    DepositBlockCreated(currentChildBlock, root, txBytes);
```

Comment:
This is a great way to ensure the deposit transaction is broadcasted. Not sure if we have to include the root again though.

Reply:
I added it so I can verify (or use directly without calculating it again). We can remove it, no problem.